### ISSUE TITLE
Shift clicking items into shulker box deletes items

### ISSUE THREAD

**Issue by Folas1337** [Phoenix616/OpenShulkerBoxes#2]
#### Used Version
OpenShulkerBoxes version 1.0.1-SNAPSHOT (build 27)
 
#### Config
Basically default config, the same as I used in #1 
 
#### Environment description
This server is running Paper version git-Paper-81 (MC: 1.19) (Implementing API version 1.19-R0.1-SNAPSHOT) (Git: 86f87ba)
 
 
#### Full Log
I hope there are no logs required for this

#### What other programs/plugins are you running?
Lots as it was discovered on a production server but I'm assuming it's very related to #1 

#### What is happening?
Here's a video showing it: https://youtu.be/nARz0PMqamg
Basically items that are shift-clicking INTO the shulker now disappear, before it was items that were shift-clicked OUT of the shulker box.

#### What did you expect to happen?
No deletion of items.

**Comment by Folas1337** [comment]
Any update on this or at least an idea why it's happening and if you can prevent it?
I know this is probably a pain to deal with but I'd at least like to know if there will ever be something done about it, if it just takes a while it's absolutely fine and understandable :)

**Comment by Phoenix616** [comment]
Unfortunately shift-click isn't really an easy to work with thing in the api but now it should at least not delete items anymore. It will only move items into empty slots and not merge existing item stacks though but that's the best I can do for now as the underlying logic doesn't really like cases where multiple/existing slots are edited and the plugin api doesn't expose that well.

**Comment by Folas1337** [comment]
Totally fine, I fully understand. The main goal is to make items not get deleted anymore and you've achieved that, at least I hope so.
I'll be testing it tomorrow but I'm guessing it'll be fine :) Thanks for the quick update once again!
